app/Models/Bible/BibleFileset.php

```diff
@@ -194,6 +194,15 @@ public static function filesetTypeTextPlainAssociated(?string $fileset_id) : Bib
             return $fileset;
         }
 
+        // E.g filset_associated = NT and tex plain = NTP
```

Comment:
filset change to fileset
tex change to text

Reply:
@bradflood , it has been fixed. https://github.com/faithcomesbyhearing/dbp/pull/985/files#diff-c4f12b64765ea311cd0953e83e7e57a88eb1d01ff5ceb4d584a163f5c949e294R197